cli/cmd/stat.go

```diff
@@ -142,7 +147,7 @@ func displayStats(resp *pb.MetricResponse, w *tabwriter.Writer) {
 	for _, name := range sortedNames {
 		fmt.Fprintf(
 			w,
-			"%s\t%frps\t%f%%\t%dms\t%dms\n",
+			"%s\t%.2frps\t%.2f%%\t%dms\t%dms\n",
```

Comment:
I think the request rate only needs a single digit of precision -- if we only get one request in a window, the smallest value we can represent is `0.1`

Also, I'd really really like these to be right-aligned (i.e. in the column).i'd be happy to address this in a followup if you'd prefer, though.

Reply:
Cool, all makes sense. I took a stab at the alignment in 84ce6959. Looks like this:

```
NAME            REQUEST_RATE   SUCCESS_RATE   P50_LATENCY   P99_LATENCY
default/hello         7.5rps        100.00%           0ms          32ms
default/world         7.7rps        100.00%           0ms          12ms
```